pavelib/assets.py

```diff
@@ -49,6 +49,7 @@
     'underscore.string/dist/underscore.string.js',
     'picturefill/dist/picturefill.js',
     'backbone/backbone.js',
+    'backbone.paginator/lib/backbone.paginator.js',
```

Comment:
Ideally since backbone.paginator is a library used internally by edx-ui-toolkit, there shouldn't be a need to either install it or copy it from node_modules. I think this should be bundled into edx-ui-toolkit and by including it would automatically include backbone.paginator.